tests/Rector/ClassMethod/ActionSuffixRemoverRector/Fixture/fixture2.php.inc

```diff
@@ -21,7 +21,7 @@ use Rector\Symfony\Tests\Rector\Source\AbstractSymfonyController;
 
 class AppController2 extends AbstractSymfonyController
 {
-    public function index()
+    public function indexAction()
```

Comment:
Also, it would be easier to duplicate one or converted fixture and add `static` there.

Reply:
> Also, it would be easier to duplicate one or converted fixture and add `static` there.

You mean, that adding for fixtures for this test is enough, since it indirectly tests ControllerMethodAnalyzer?
I did just refactor my TestCase from the PR use the container, but I still have to create the Nodes manually.

Reply:
> You mean, that adding for fixtures for this test is enough, since it indirectly tests ControllerMethodAnalyzer

Exactly! 👍